flow/SystemMonitor.cpp

```diff
@@ -67,37 +61,32 @@ SystemStatistics getSystemStatistics() {
 	    .detail("ApproximateUnusedMemory" #size, FastAllocator<size>::getApproximateMemoryUnused())                    \
 	    .detail("ActiveThreads" #size, FastAllocator<size>::getActiveThreads())
 
-SystemStatistics customSystemMonitorNetworkBusyness(std::string eventName, StatisticsState* statState) {
-	// Custom system monitor to update the network busyness on a 1s cadence
+// update the network busyness on a 1s cadence
+void monitorNetworkBusyness() {
 	const IPAddress ipAddr = IPAddress();
-	SystemStatistics currentStats = getSystemStatistics("", &ipAddr, &statState->systemState, true);
-	NetworkData netData;
-	netData.init();
+	static StatisticsState statState = StatisticsState();
+	SystemStatistics currentStats = getSystemStatistics("", &ipAddr, &(&statState)->systemState, true);
```

Comment:
It looks like the only thing we're using out of here is the elapsed time, and we can capture that more directly. Possibly the easiest approach would be to write this as its own ACTOR with a loop that looks like:

```
state double lastTime = now();
loop {
    wait(delay(UPDATE_INTERVAL), TaskPriority::SomeHighPriority);
    double elapsed = now() - lastTime;
    lastTime = now();
    // update metrics
}
```